spec/javascripts/tests/MultiTableFilter_spec.js

```diff
@@ -0,0 +1,88 @@
+describe('filter table based on text field criteria', function () {
+  'use strict';
+
+  function visibleRows() {
+    return $('.js-filter-rows').find('tr:visible');
+  };
+
+  function setFilterText(text) {
+    $('.js-filter-input').val(text);
+    $('.js-filter-input').change();
+  };
+
+  describe('on a table', function() {
+    beforeEach(function (done) {
+      var self = this;
+
+      requirejs(['jquery', 'jqueryFastLiveFilter', 'MultiTableFilter'], function ($, jqueryFastLiveFilter, MultiTableFilter) {
```

Comment:
As I understand it: the dependencies are loaded by the client, but they won't be available in the scope of the function — since requirejs avoids globals and only gives you things when you ask for them to be direct injected for you. 

Either way, the tests fail with that removed — though it's possible that the syntax might need changing in some other way I'm not aware of...